src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitorContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+
+namespace Microsoft.PowerFx.Interpreter
```

Comment:
EvalVisitor is in the Microsoft.PowerFx namespace. We should make this class consistent with EvalVisitor.

Reply:
Is now consistent with EvalVisitor